druid-shell/src/window.rs

```diff
@@ -17,6 +17,8 @@
 use std::any::Any;
 use std::ops::Deref;
 
+use crate::clipboard::ClipboardItem;
+//TODO: why is this pub?
```

Comment:
I think we need to clean this up a bit :/. I don't have a super-clear picture of exactly what these types should be. Seems out of scope for this PR though.

Reply:
yea, there's a bunch of related cleanup, I'll add this to the list.